Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt

```diff
@@ -205,11 +206,17 @@ fun Messages(
         LazyColumn(
             reverseLayout = true,
             state = scrollState,
+            // Add content padding so that the content can be scrolled (y-axis)
+            // below the status bar + app bar
+            // TODO: Get height from somewhere
+            contentPadding = LocalWindowInsets.current.statusBars.toPaddingValues(
+                additionalTop = 90.dp
+            ),
             modifier = Modifier
                 .testTag(ConversationTestTag)
-                .fillMaxWidth()
+                .fillMaxSize()
         ) {
-            items(messages.size) { index ->
+            for (index in messages.indices) {
```

Comment:
`forEachIndexed`?

Reply:
Could do, but it doesn't really add much here imo. The code below does multiple lookups based on the index.